week1/main.py

```diff
@@ -16,7 +16,40 @@ def print_board(board):
 
 # complete the logic to find the winner
 def check_winner(board):
-   return False
+    if board[0][0] == 'X' and board[0][1] == 'X' and board[0][2] == 'X':
```

Comment:
good job. 

```suggestion
    if board[0][0] == board[0][1] == 'X' == board[0][2] and board[0][0] != ' ':
```

one slight improvement that you can explore to avoid doing the same logic for both X and O